client/js/modules.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
```

Comment:
Why is this stuff not just in utils?

Reply:
~I agree.~ Actually, all those are just used once, I think this entire file/refactor is not wanted.